app/models/playlist.rb

```diff
@@ -5,12 +5,13 @@ class Playlist < ActiveRecord::Base
 
   scope :albums,           -> { where(is_mix: false).where(is_favorite: false) }
   scope :favorites,        -> { where(is_favorite: true) }
-  scope :for_home,         -> { select('distinct playlists.user_id, playlists.*').recent.only_public.with_preloads }
+  scope :for_home,         -> { select('distinct playlists.user_id, playlists.*').recently_published.only_public.with_preloads }
   scope :include_private,  -> { where(is_favorite: false) }
   scope :mixes,            -> { where(is_mix: true) }
   scope :only_public,      -> { where(private: false).where(is_favorite: false).where("tracks_count > 1") }
```

Comment:
Seems like`where("tracks_count > 1")` is no longer needed here if we are guaranteeing a playlist can't be published unless it has 2 tracks.